source/minimessage.rst

```diff
@@ -152,10 +134,12 @@ Color (2, hex/rgb)
 
 A different, more flexible way (supports hex colors!) for colors looks like this
```

Comment:
phrased like this method supports hex colors while the other method does not, and since they both seem to support both named colors and hex colors I don't think the second one is more "flexible".